### ISSUE TITLE
Split settings file into separate settings for dev and production environment

### ISSUE THREAD

**Issue by hamaMk** [terrameijar/invoices#4]
Separating settings allows for easy handling of configurations based on the environment e.g. SQLite database connection in development and Postgres in production.

**Comment by hamaMk** [comment]
Yes first of all we replace the standard settings file with a module, then we create a base.py file that will contain common settings. We can now go on to create two more files dev.py and prod.py that will extend base.py. Voila.

**Comment by terrameijar** [comment]
Understood @hamaMk , please work on it and make a PR once you're done

> Yes first of all we replace the standard settings file with a module, then we create a base.py file that will contain common settings. We can now go on to create two more files dev.py and prod.py that will extend base.py. Voila.
